Include/internal/pycore_code.h

```diff
@@ -473,12 +475,12 @@ adaptive_counter_cooldown(void) {
 
 static inline uint16_t
 adaptive_counter_backoff(uint16_t counter) {
-    unsigned int backoff = counter & ((1<<ADAPTIVE_BACKOFF_BITS)-1);
+    uint16_t backoff = counter & ((1 << ADAPTIVE_BACKOFF_BITS) - 1);
```

Comment:
Is there difference between `1 << ADAPTIVE_BACKOFF_BITS` and `1u << ADAPTIVE_BACKOFF_BITS`?

Reply:
I hesitated to replace 1 with 1U. With proposed code, there is no warning. With 1U I got additional warnings 🤷🏻‍♂️ Maybe because "unsigned int" is larger than uint16_t. I don't know rules of implicit cast in C.